2/Dockerfile

```diff
@@ -91,6 +91,10 @@ RUN /usr/local/bin/install-plugins.sh /opt/openshift/base-plugins.txt && \
     /usr/local/bin/fix-permissions /var/lib/jenkins && \
     /usr/local/bin/fix-permissions /var/log
 
+# Set default git configuration
+RUN git config --global user.email "openshift@openshift.com" && \
+    git config --global user.name "openshift"
```

Comment:
please append these commands to the existing RUN command above, so it doesn't create a new layer.

Reply:
good idea